src/main/js/page_objects/blueocean/bluePipelineRunDetail.js

```diff
@@ -110,6 +110,19 @@ module.exports.commands = [{
         // E.g. nothing on the pipeline graph that allows us to find it.
         return this;
     },
+    assertMultiBranchResult: function (callback) {
```

Comment:
Actually that part it's a must. That belongs in the detail page. @tfennelly will surely agree